src/app/content/components/Page/PageComponent.tsx

```diff
@@ -149,6 +147,7 @@ export default class PageComponent extends Component<PagePropTypes> {
     return <MinPageHeight>
       <this.highlightManager.CardList />
       <PT />
+      <MediaModalPortal />
```

Comment:
if you look above, the CardList is coming from the manager so that you don't need all the wonky binding logic. if you prefer, you could leave the component separate and pass the manager into it as a prop here, but either of those approaches would be more clear than the singleton binding its doing now